creating_diverse_inclusive_communities.adoc

```diff
@@ -66,7 +66,7 @@ And this data was collected pre-2020.
 
 This year's confluence of a global pandemic and mass civil unrest has led to increased attacks against people of color, specifically Asian Americans and Black Americans.
 If attackers feel emboldened to hurt people in the flesh, it's no surprise that this https://www.nytimes.com/2020/07/11/business/media/tucker-carlson-writer-blake-neff.html[occurs online] as well.
-If you work on open source projects with Black and Asian contributors, make no mistake: They are in pain.
+If you work on OS projects with Black and Asian contributors, make no mistake: They are in pain.
```

Comment:
I'll need to ask @theopensourceway/editors, but I believe we're spelling out "open source" throughout this guidebook.